### ISSUE TITLE
Is the OCR evaluation model (ster\crnn) and tatt end-to-end?

### ISSUE THREAD

**Issue by HansonnnCheung** [mjq11302010044/TATT#8]
Is the OCR evaluation model (ster\crnn) and tatt end-to-end ?  OR first use SR model to output results, and then input OCR?
 just like the code below： 
 def __getitem__(self, index):
    ...
    ...
    label_str = str_filt(word, self.voc_type)
    return img_HR, img_lr, img_HRy, img_lry, label_str

Does “label_str” participate in the training of the whole model？

**Comment by mjq11302010044** [comment]
@HansonnnCheung Not really, TATT follows the settings in the TPGSR, i.e., the label_str is not used in training.

**Comment by HansonnnCheung** [comment]
Thank you for your answer.  Where did the “teaching_loss” come from during the training. Is "TEXT_Prior" generated by an OCR model?  @mjq11302010044

In addition, if  "lable_str"  does not participate in training, is it necessary to use the value of AAA as an input during DataLoader?

**Comment by HansonnnCheung** [comment]
Do you mean that “TPGSR and its settings”  is the TPGSR model，which published in your home page? @mjq11302010044
